src/frontend/src/pages/PIAForm/BannerStatus/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState, useEffect } from 'react';
+import { IBannerStatusProps } from './interfaces';
+import Callout from '../../../components/common/Callout';
+import { PiaStatuses } from '../../../constant/constant';
+import { isMPORole } from '../../../utils/helper.util';
+
+const BannerStatus = ({ pia }: IBannerStatusProps) => {
+  const [bannerMessage, setBannerMessage] = useState<string>('');
+  useEffect(() => {
```

Comment:
These if/else should be avoided. We have a status utility file (under utility/status) that can do this processing for us more elegantly.